Failing to install

im not sure if this is a problem that can be fixed, every time i try to install gsplat i end up with this error:
```      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec2.hpp(101): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec2.hpp(101): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec3.hpp(107): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec3.hpp(107): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec4.hpp(105): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec4.hpp(105): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x2.hpp(35): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x2.hpp(35): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x3.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x3.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x4.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x4.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x2.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x2.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x3.hpp(35): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x3.hpp(35): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\detail\type_vec1.hpp(95): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\detail\type_vec1.hpp(95): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x4.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x4.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x2.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x2.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x3.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x3.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x4.hpp(35): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x4.hpp(35): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_quat.hpp(73): warning #20012-D: __device__ annotation is ignored on a function("qua") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_quat.hpp(73): warning #20012-D: __host__ annotation is ignored on a function("qua") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\miniconda3\envs\nerfstudio\lib\site-packages\torch\include\pybind11\detail/common.h(210): warning C4005: 'HAVE_SNPRINTF': macro redefinition
      C:\Users\cmaje\miniconda3\envs\nerfstudio\include\pyerrors.h(315): note: see previous definition of 'HAVE_SNPRINTF'
      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec2.hpp(101): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec2.hpp(101): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec3.hpp(107): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec3.hpp(107): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec4.hpp(105): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec4.hpp(105): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x2.hpp(35): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x2.hpp(35): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x3.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x3.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x4.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x4.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x2.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x2.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x3.hpp(35): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x3.hpp(35): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\detail\type_vec1.hpp(95): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\detail\type_vec1.hpp(95): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x4.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x4.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x2.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x2.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x3.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x3.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x4.hpp(35): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x4.hpp(35): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_quat.hpp(73): warning #20012-D: __device__ annotation is ignored on a function("qua") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_quat.hpp(73): warning #20012-D: __host__ annotation is ignored on a function("qua") that is explicitly defaulted on its first declaration

      projection.cu
      "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.8\bin\nvcc" -c gsplat\cuda\csrc\rasterization.cu -o build\temp.win-amd64-cpython-38\Release\gsplat\cuda\csrc\rasterization.obj -Igsplat\cuda\csrc -IC:\Users\cmaje\miniconda3\envs\nerfstudio\lib\site-packages\torch\include -IC:\Users\cmaje\miniconda3\envs\nerfstudio\lib\site-packages\torch\include\torch\csrc\api\include -IC:\Users\cmaje\miniconda3\envs\nerfstudio\lib\site-packages\torch\include\TH -IC:\Users\cmaje\miniconda3\envs\nerfstudio\lib\site-packages\torch\include\THC "-IC:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.8\include" -IC:\Users\cmaje\miniconda3\envs\nerfstudio\include -IC:\Users\cmaje\miniconda3\envs\nerfstudio\Include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\cppwinrt" -Xcudafe --diag_suppress=dll_interface_conflict_dllexport_assumed -Xcudafe --diag_suppress=dll_interface_conflict_none_assumed -Xcudafe --diag_suppress=field_without_dll_interface -Xcudafe --diag_suppress=base_class_has_different_dll_interface -Xcompiler /EHsc -Xcompiler /wd4068 -Xcompiler /wd4067 -Xcompiler /wd4624 -Xcompiler /wd4190 -Xcompiler /wd4018 -Xcompiler /wd4275 -Xcompiler /wd4267 -Xcompiler /wd4244 -Xcompiler /wd4251 -Xcompiler /wd4819 -Xcompiler /MD -D__CUDA_NO_HALF_OPERATORS__ -D__CUDA_NO_HALF_CONVERSIONS__ -D__CUDA_NO_BFLOAT16_CONVERSIONS__ -D__CUDA_NO_HALF2_OPERATORS__ --expt-relaxed-constexpr -O3 --use_fast_math --expt-relaxed-constexpr -DWIN32_LEAN_AND_MEAN -DTORCH_API_INCLUDE_EXTENSION_H -DTORCH_EXTENSION_NAME=csrc -D_GLIBCXX_USE_CXX11_ABI=0 -gencode=arch=compute_89,code=compute_89 -gencode=arch=compute_89,code=sm_89 -std=c++17 --use-local-env
      C:\Users\cmaje\miniconda3\envs\nerfstudio\lib\site-packages\torch\include\pybind11\detail/common.h(210): warning C4005: 'HAVE_SNPRINTF': macro redefinition
      C:\Users\cmaje\miniconda3\envs\nerfstudio\include\pyerrors.h(315): note: see previous definition of 'HAVE_SNPRINTF'
      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec2.hpp(101): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec2.hpp(101): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec3.hpp(107): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec3.hpp(107): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec4.hpp(105): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec4.hpp(105): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x2.hpp(35): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x2.hpp(35): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x3.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x3.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x4.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x4.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x2.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x2.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x3.hpp(35): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x3.hpp(35): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\detail\type_vec1.hpp(95): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\detail\type_vec1.hpp(95): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x4.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x4.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x2.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x2.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x3.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x3.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x4.hpp(35): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x4.hpp(35): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_quat.hpp(73): warning #20012-D: __device__ annotation is ignored on a function("qua") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_quat.hpp(73): warning #20012-D: __host__ annotation is ignored on a function("qua") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\miniconda3\envs\nerfstudio\lib\site-packages\torch\include\pybind11\detail/common.h(210): warning C4005: 'HAVE_SNPRINTF': macro redefinition
      C:\Users\cmaje\miniconda3\envs\nerfstudio\include\pyerrors.h(315): note: see previous definition of 'HAVE_SNPRINTF'
      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec2.hpp(101): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec2.hpp(101): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec3.hpp(107): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec3.hpp(107): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec4.hpp(105): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec4.hpp(105): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x2.hpp(35): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x2.hpp(35): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x3.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x3.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x4.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x4.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x2.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x2.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x3.hpp(35): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x3.hpp(35): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\detail\type_vec1.hpp(95): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\detail\type_vec1.hpp(95): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x4.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x4.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x2.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x2.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x3.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x3.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x4.hpp(35): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x4.hpp(35): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_quat.hpp(73): warning #20012-D: __device__ annotation is ignored on a function("qua") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_quat.hpp(73): warning #20012-D: __host__ annotation is ignored on a function("qua") that is explicitly defaulted on its first declaration

      rasterization.cu
      "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.8\bin\nvcc" -c gsplat\cuda\csrc\relocation.cu -o build\temp.win-amd64-cpython-38\Release\gsplat\cuda\csrc\relocation.obj -Igsplat\cuda\csrc -IC:\Users\cmaje\miniconda3\envs\nerfstudio\lib\site-packages\torch\include -IC:\Users\cmaje\miniconda3\envs\nerfstudio\lib\site-packages\torch\include\torch\csrc\api\include -IC:\Users\cmaje\miniconda3\envs\nerfstudio\lib\site-packages\torch\include\TH -IC:\Users\cmaje\miniconda3\envs\nerfstudio\lib\site-packages\torch\include\THC "-IC:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.8\include" -IC:\Users\cmaje\miniconda3\envs\nerfstudio\include -IC:\Users\cmaje\miniconda3\envs\nerfstudio\Include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\cppwinrt" -Xcudafe --diag_suppress=dll_interface_conflict_dllexport_assumed -Xcudafe --diag_suppress=dll_interface_conflict_none_assumed -Xcudafe --diag_suppress=field_without_dll_interface -Xcudafe --diag_suppress=base_class_has_different_dll_interface -Xcompiler /EHsc -Xcompiler /wd4068 -Xcompiler /wd4067 -Xcompiler /wd4624 -Xcompiler /wd4190 -Xcompiler /wd4018 -Xcompiler /wd4275 -Xcompiler /wd4267 -Xcompiler /wd4244 -Xcompiler /wd4251 -Xcompiler /wd4819 -Xcompiler /MD -D__CUDA_NO_HALF_OPERATORS__ -D__CUDA_NO_HALF_CONVERSIONS__ -D__CUDA_NO_BFLOAT16_CONVERSIONS__ -D__CUDA_NO_HALF2_OPERATORS__ --expt-relaxed-constexpr -O3 --use_fast_math --expt-relaxed-constexpr -DWIN32_LEAN_AND_MEAN -DTORCH_API_INCLUDE_EXTENSION_H -DTORCH_EXTENSION_NAME=csrc -D_GLIBCXX_USE_CXX11_ABI=0 -gencode=arch=compute_89,code=compute_89 -gencode=arch=compute_89,code=sm_89 -std=c++17 --use-local-env
      C:\Users\cmaje\miniconda3\envs\nerfstudio\lib\site-packages\torch\include\pybind11\detail/common.h(210): warning C4005: 'HAVE_SNPRINTF': macro redefinition
      C:\Users\cmaje\miniconda3\envs\nerfstudio\include\pyerrors.h(315): note: see previous definition of 'HAVE_SNPRINTF'
      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec2.hpp(101): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec2.hpp(101): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec3.hpp(107): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec3.hpp(107): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec4.hpp(105): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_vec4.hpp(105): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x2.hpp(35): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x2.hpp(35): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x3.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x3.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x4.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat2x4.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x2.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x2.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x3.hpp(35): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x3.hpp(35): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\detail\type_vec1.hpp(95): warning #20012-D: __device__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\detail\type_vec1.hpp(95): warning #20012-D: __host__ annotation is ignored on a function("vec") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x4.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat3x4.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x2.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x2.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x3.hpp(36): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x3.hpp(36): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x4.hpp(35): warning #20012-D: __device__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_mat4x4.hpp(35): warning #20012-D: __host__ annotation is ignored on a function("mat") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_quat.hpp(73): warning #20012-D: __device__ annotation is ignored on a function("qua") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\third_party\glm\glm\ext\../detail/type_quat.hpp(73): warning #20012-D: __host__ annotation is ignored on a function("qua") that is explicitly defaulted on its first declaration

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\relocation.cu(29): error: calling a __host__ function("double  ::sqrt<int, (int)0> (T1)") from a __global__ function("compute_relocation_kernel") is not allowed

      C:\Users\cmaje\AppData\Local\Temp\pip-req-build-axl_ri12\gsplat\cuda\csrc\relocation.cu(29): error: identifier "sqrt<int, (int)0> " is undefined in device code

      2 errors detected in the compilation of "gsplat/cuda/csrc/relocation.cu".
      relocation.cu
      error: command 'C:\\Program Files\\NVIDIA GPU Computing Toolkit\\CUDA\\v11.8\\bin\\nvcc.exe' failed with exit code 1
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for gsplat
  Running setup.py clean for gsplat
Failed to build gsplat
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (gsplat)```

--- a/gsplat/cuda/csrc/relocation.cu
+++ b/gsplat/cuda/csrc/relocation.cu
@@ -1,16 +1,9 @@
 #include "bindings.h"
 
 // Equation (9) in "3D Gaussian Splatting as Markov Chain Monte Carlo"
-__global__ void compute_relocation_kernel(
-    int N,
-    float *opacities,
-    float *scales,
-    int *ratios,
-    float *binoms,
-    int n_max,
-    float *new_opacities,
-    float *new_scales
-) {
+__global__ void compute_relocation_kernel(int N, float *opacities, float *scales,
+                                          int *ratios, float *binoms, int n_max,
+                                          float *new_opacities, float *new_scales) {
     int idx = threadIdx.x + blockIdx.x * blockDim.x;
     if (idx >= N)
         return;
@@ -25,8 +18,8 @@ __global__ void compute_relocation_kernel(
     for (int i = 1; i <= n_idx; ++i) {
         for (int k = 0; k <= (i - 1); ++k) {
             float bin_coeff = binoms[(i - 1) * n_max + k];
-            float term =
-                (pow(-1, k) / sqrt(k + 1)) * pow(new_opacities[idx], k + 1);
+            float term = (pow(-1.0f, k) / sqrt(static_cast<float>(k + 1))) *
+                         pow(new_opacities[idx], k + 1);
             denom_sum += (bin_coeff * term);
         }
     }
@@ -35,13 +28,10 @@ __global__ void compute_relocation_kernel(
         new_scales[idx * 3 + i] = coeff * scales[idx * 3 + i];
 }
 
-std::tuple<torch::Tensor, torch::Tensor> compute_relocation_tensor(
-    torch::Tensor &opacities,
-    torch::Tensor &scales,
-    torch::Tensor &ratios,
-    torch::Tensor &binoms,
-    const int n_max
-) {
+std::tuple<torch::Tensor, torch::Tensor>
+compute_relocation_tensor(torch::Tensor &opacities, torch::Tensor &scales,
+                          torch::Tensor &ratios, torch::Tensor &binoms,
+                          const int n_max) {
     DEVICE_GUARD(opacities);
     CHECK_INPUT(opacities);
     CHECK_INPUT(scales);
@@ -53,16 +43,11 @@ std::tuple<torch::Tensor, torch::Tensor> compute_relocation_tensor(
     uint32_t N = opacities.size(0);
     if (N) {
         at::cuda::CUDAStream stream = at::cuda::getCurrentCUDAStream();
-        compute_relocation_kernel<<<(N + N_THREADS - 1) / N_THREADS, N_THREADS, 0, stream>>>(
-            N,
-            opacities.data_ptr<float>(),
-            scales.data_ptr<float>(),
-            ratios.data_ptr<int>(),
-            binoms.data_ptr<float>(),
-            n_max,
-            new_opacities.data_ptr<float>(),
-            new_scales.data_ptr<float>()
-        );
+        compute_relocation_kernel<<<(N + N_THREADS - 1) / N_THREADS, N_THREADS, 0,
+                                    stream>>>(
+            N, opacities.data_ptr<float>(), scales.data_ptr<float>(),
+            ratios.data_ptr<int>(), binoms.data_ptr<float>(), n_max,
+            new_opacities.data_ptr<float>(), new_scales.data_ptr<float>());
     }
     return std::make_tuple(new_opacities, new_scales);
 }

